### PR TITLE
doc: update Quickstart's step2 to try the getting started guide in README_CN.md

### LINKED ISSUE #1289: doc: Chinese version
This is the English version, we also need to update the Chinese version.

https://github.com/apache/incubator-apisix/pull/1287

### PR DESCRIPTION
**Pull request by dabue**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #1289

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -132,9 +132,9 @@ CentOS 7, Ubuntu 16.04, Ubuntu 18.04, Debian 9, Debian 10, macOS, **[ARM64](http
 sudo apisix start
 ```
 
-2. 测试限流插件
+2. 入门指南
 
-你可以测试限流插件，来上手体验 APISIX，按照[限流插件文档](doc/plugins/limit-count-cn.md)的步骤即可。
+入门指南是学习APISIX基础知识的好方法。按照 [入门指南](doc/getting-started.md)的步骤即可。
```
Comment (membphis) [inline README_CN.md]: need a space between Chinese and English

Comment (dabue) [reply README_CN.md]: I got it.

Comment (dabue) [reply README_CN.md]: I have updated it.

**Comment by membphis** [comment]
merged, many thx

**Pull request merged**